.github/README.md

```diff
@@ -0,0 +1,27 @@
+# Bridge-In-Tech (android)
+
+
+## Contributing
+
+**This project is under active development**
+
+Bridge-In-Tech is an extension of Mentorship System that allows industry companies, mentors and students to actively contribute to one another. 
+
+This is the Android client of Bridge-In-Tech.
+
+Please read our [Contributing Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/REPORTING_GUIDELINES.md) thoroughly.
+
+
+## Branches
+
+This repository has the following branches:
+- **master**: This branch contains the deployment of the backend.
+- **develop**: This contains the latest code. All the contributing PRs must be sent to this branch.
+
+## Contact
+
+If you have any questions or want to discuss something about this repo, feel free to join our [Zulip Community](http://anitab-org.zulipchat.com/)! If you are a new contributor, head over to this project's stream [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech) on Zulip to see ongoing discussions.
+
+## License
+
+The project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](https://github.com/anitab-org/anitab-org.github.io/blob/develop/LICENSE) file.
```

Comment:
```suggestion
The project is licensed under the GNU General Public License v3.0. Learn more about it in the [LICENSE](LICENSE) file.
```

Relative path, so this always links to the correct file. For example on your fork, GitHub will be able to redirect to the file on your fork repo :)

Reply:
> I also wouldn't say it is an extension of the mentorship system, the purpose is different than mentorship system. Right?

My bad, I think I'm to blame for this word 'extension' coz I used this in my wiki home page which is why probably @Menina294 use it here. Sorry again. I'll fix the homepage now 😆

Reply:
No problem, I actually think that your idea is one of its own. Maybe I am seeing this from a wrong angle @mtreacy002 if you say so, I don't mind :)